CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # splat Release Notes
 
+## 0.9.0: The Big Update
+### Introducing [spimdisasm](https://github.com/Decompollaborate/spimdisasm)!
+* Thanks to [AngheloAlf](https://github.com/AngheloAlf), we now have a much better MIPS disassembler in splat! spimdisasm has much better hi/lo matching, much lower ram usage, and is just overall grood.
+
+We plan to roll this out in phases. Currently, it only handles actual code disassembly. Later on, we will probably migrate our current data assembly code to use spimdisasm as well.
+
+**NOTICE**: This integration has been tested on a variety of games and configurations. However, with any giant change to the platform like this, there are bound to be things we didn't catch. Please be patient with us as we handle these remaining issues. Though from what we've seen already, the slight bugs one may come across are totally worth the much improved diassembly.
+
+### gfx segment type
+* A new `gfx` segment type is available, which create a c file containing a disassembled display list according to the segment's start and end offsets. Thanks to [Glank](https://github.com/glankk) and [Tharo](https://github.com/thar0/) for their work on [libgfxd](https://github.com/glankk/libgfxd) and [pygfxd](https://github.com/thar0/pygfxd/), respectively, for helping make this a possibility in splat.
+
+### API breaking changes
+* Some Segment() arguments have changed, which may cause extensions to break. Please see the `__init__` function for Segment for more details.
+
+### symbol_addrs.txt changes
+* symbol_addrs now supports the `segment:` attribute, which allows specifying the symbol's top-level segment. This can be helpful for symbol resolution when there are overlapping overlays. See `exclusive_ram_id` below for more information.
+
+### Global options changes
+
+The new `symbol_name_format` option allows specification of how symbols will be named. This can be set as a global option and also changed per-segment. `symbol_noram_name_format` is used when the symbol does not have a rom address (BSS).
+
+ The following substitutions are allowed:
+
+`$ROM` - the rom address of the symbol, hex-formatted (omitted for BSS symbols)
```

Comment:
for symbols in noload sections, does this get expanded to an empty string or error?

It may be worth giving an example for each of these substitutions, as it could be `0xabc`, `0xABC`, `0Xabc`, `0XABC`, `0x000abc`, `0x000ABC`, `0X000abc`, `0X000ABC`, `abc`, `ABC`, `abc`, `ABC`, `000abc`, `000ABC`, `000abc`, `000ABC` :smiley_cat:

Reply:
If one tries to use $ROM in the "norom" format, for example, they'll get a big error that will explain the issue